dashboard/engines/marketing/lib/marketing/engine.rb

```diff
@@ -0,0 +1,5 @@
+module Marketing
+  class Engine < ::Rails::Engine
+    isolate_namespace Marketing
```

Comment:
Please add this configuration to autoload files from the lib directory, so we would not need to require them manually anymore:
```suggestion
    isolate_namespace Marketing
    
    config.autoload_paths << config.root.join('lib').to_s
```

Reply:
https://github.com/code-dot-org/code-dot-org/blob/6c876dc7caa0389d8981e388d028e9a00f4ca395/dashboard/config/application.rb#L189